DEPS

```diff
@@ -103,6 +103,10 @@ vars = {
   # Checkout Linux dependencies only when building on Linux.
   'download_linux_deps': 'host_os == "linux"',
 
+  # The minimum macOS SDK version. This must match the setting in
+  # //build/config/mac/mac_sdk.gni.
+  'mac_sdk_min': '10.14',
```

Comment:
If the min sdk is also declared here, does it also need to be in buildroot? Does `//build/config/mac/mac_sdk.gni` need to exist?

Reply:
Before this PR and the buildroot PR, this value was in two places. Once in the `gn` script, and once in `mac_sdk.gni`. The instance in `mac_sdk.gni` is unnecessary with some slight rearrangement of the logic in the `gn` script. The value in the `gn` script could go away too, if I introduce parsing of the `DEPS` file into the `gn` script, but that might be overkill for this PR.

Reply:
Should "This must match the setting in //build/config/mac/mac_sdk.gni" comment be updated to reference `tools/gn`, then?

https://github.com/flutter/buildroot/commit/ebc2748229c8377d97ee1cafd11dff07c285f440#diff-128083755b38060af3a265f785fd4488d7ac6feaad498a1ee0a8b7cdd8e330c3R10

Reply:
Done.